docs/src/components/ColorSwatchFamily/ColorSwatchFamily.component.tsx

```diff
@@ -0,0 +1,57 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { ColorSwatch } from '../ColorSwatch';
+import { Typography } from '../../../../src/Typography';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyProps {
+  baseColorHexCode: string;
+  baseColorName: string;
+  darkColorHexCode?: string;
+  lightColorHexCode?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamily = styled.div`
+  margin: 1rem 0;
+`;
+
+
+// JSX MARKUP
+export const ColorSwatchFamily = ({ baseColorHexCode, baseColorName, darkColorHexCode, lightColorHexCode }: ColorSwatchFamilyProps): React.ReactElement<any> => (
+  <StyledColorSwatchFamily
+    baseColorName={baseColorName}
+    baseColorHexCode={baseColorHexCode}
+    darkColorHexCode={darkColorHexCode}
+    lightColorHexCode={lightColorHexCode}
+  >
+
+    <Typography as="h2" weight="600">{baseColorName}</Typography>
+
+    <ColorSwatch
+      colorHexCode={baseColorHexCode}
+      colorName={baseColorName+'.base'}
```

Comment:
```
colorName={`${baseColorName}.base`}
```

Reply:
[sauce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
The sauce was a fascinating read / sumptuous flavor journey 🍝